build-info-extractor/src/test/java/org/jfrog/build/extractor/clientConfiguration/util/spec/SpecsHelperIntegrationTest.java

```diff
@@ -34,11 +31,28 @@
     private static final String INTEGRATION_TESTS = "/integration/tests";
     private static final String DEFAULT_SPEC_PATH = "/integration/default";
 
+    @BeforeClass
+    public void init() throws IOException {
+        super.init();
+        if (!dependenciesClient.isArtifactoryOSS()) {
+            createTestRepo(localRepo2);
+        }
+    }
+
+    @AfterClass
+    protected void terminate() throws IOException {
+        if (!dependenciesClient.isArtifactoryOSS()) {
+            deleteTestRepo(localRepo2);
+        }
+        super.terminate();
+    }
+
     @BeforeMethod
     @AfterMethod
     protected void cleanup() throws IOException {
         FileUtils.deleteDirectory(tempWorkspace);
         deleteContentFromRepo(localRepo);
```

Comment:
localRepo --> localRepo1

(since we now added localRepo2)

Reply:
Changed.